### PR TITLE
Add comments for readability

### PR DESCRIPTION
**Pull request by yunjey**
Added comments for index_ignore arguments in cross_entropy loss function. Also copied comments for the arguments in nn.NLLLoss and pasted them into nn.CrossEntropy. This will help users to understand nn.CrossEntropy easily.

### CODE REVIEW

**Comment by apaszke** [comment]
The liner has some complaints. You need to remove whitespace from empty lines

**Comment by yunjey** [comment]
@apaszke Thanks. Now, i removed whitespace from the empty lines.

**Review by apaszke — APPROVED** [review]

**Comment by apaszke** [comment]
Thanks!
